scripts/download_shellcheck.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/bash
```

Comment:
Since we already have a script ( ./entrypoint.sh ) at the repo's root. I think we better not have a directory called scripts at the root. The question will be what's the difference between entrypoint.sh script and the script inside the `scripts` directory. How about renaming the `scripts` directory to `src` and we'll store the supporting files there. We already have docs, but I'm not sure if that's a good idea to put this script there.